From: Ratchanan Srirattanamet <ratchanan@ubports.com>
Date: Mon, 28 Oct 2019 20:27:09 +0700
Subject: Gstreamer: fix camerabin's supported focus point mode

Commit be7fef656a1d087d3d1d3fa102da4fce85855935 (Add support for face
detection focus point mode to camerabin backend.) updated camerabin's
isFocusPointModeSupported() to indicates face detection mode support.
However, it forgets to remove the old code that says only auto and
custom is supported, making the face detection never reported as
supported.

Change-Id: I76627e0b72fb94cd1370bc990edd6c748086a5d9
Forwarded: no
Last-Update: 2019-10-28
---
 src/plugins/gstreamer/camerabin/camerabinfocus.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/plugins/gstreamer/camerabin/camerabinfocus.cpp b/src/plugins/gstreamer/camerabin/camerabinfocus.cpp
index d4e7fa69..a9cdf8e6 100644
--- a/src/plugins/gstreamer/camerabin/camerabinfocus.cpp
+++ b/src/plugins/gstreamer/camerabin/camerabinfocus.cpp
@@ -192,8 +192,6 @@ void CameraBinFocus::setFocusPointMode(QCameraFocus::FocusPointMode mode)
 
 bool CameraBinFocus::isFocusPointModeSupported(QCameraFocus::FocusPointMode mode) const
 {
-    return mode == QCameraFocus::FocusPointAuto || mode == QCameraFocus::FocusPointCustom;
-
     switch (mode) {
     case QCameraFocus::FocusPointAuto:
     case QCameraFocus::FocusPointCustom:
-- 
2.17.1

